### DISCUSSION TITLE
District Data Schwyz

### DISCUSSION THREAD

**Discussion by balzamas** [openZH/covid_19#1602] [Q&A]
I didn't want to open an issue as I am not sure if it is one:

Looking at the column TotalConfCases in
https://github.com/openZH/covid_19/blob/master/fallzahlen_bezirke/fallzahlen_kanton_SZ_bezirk.csv
I see that the numbers are going up and down. So far I understood this column always as "Total positive tests since the begin" so it only goes down when the data is corrected by the canton, but this seems not to be the case here as it's going up and down all the time.

So what else are those numbers? The source (e.g. https://www.sz.ch/public/upload/assets/51141/Corona_Fallzahlen_21_01_2021.pdf) does not help either. 

Does the canton of Schwyz estimated the currently infected persons, which seems odd but who knows? Shouldn't it be then go into another column?

Thank you for your inputs
David

**Comment by amslera** [comment]
#1620
